### PR TITLE
[1LP][RFR] Added support of Compliances for Middleware Servers. Added new tests.

### PR DESCRIPTION
**Pull request by hhovsepy**
{{pytest: cfme/tests/middleware/test_middleware_server_compliance.py -v --use-provider hawkular}}

### CODE REVIEW

**Comment by quarckster** [comment]
One test failed in 5.9

**Comment by hhovsepy** [comment]
@quarckster this failure is because of middleware environment issue, runs on my local.

File: cfme/middleware/server.py [hunk 1]
```diff
@@ -275,6 +280,32 @@ def is_stopped(self):
         self.load_details(refresh=True)
         return self.get_detail("Properties", "Server State").lower() == 'stopped'
 
+    @variable(alias='ui')
```
Comment (izapolsk) [inline cfme/middleware/server.py]: it makes sense to implement this using sentaku

File: cfme/tests/middleware/test_middleware_server_compliance.py [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+    get_hawkular_server,
+    get_domain_server,
+    verify_server_compliant
+)
+
+
+pytestmark = [
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.uncollectif(lambda: current_version() < '5.9'),
+    pytest.mark.provider([HawkularProvider], scope="function"),
```
Comment (izapolsk) [inline cfme/tests/middleware/test_middleware_server_compliance.py]: Q: are you sure you really need to add a provider for each new test ?

File: cfme/middleware/server.py (hunk 1 above)
Comment (psav) [reply cfme/middleware/server.py]: It does and we will, but right now middleware has a lot of variant and we'll convert them next year

File: cfme/tests/middleware/test_middleware_server_compliance.py (hunk 2 above)
Comment (psav) [reply cfme/tests/middleware/test_middleware_server_compliance.py]: Unless it gets removed it won't get re-added

**Pull request merged**